### ISSUE TITLE
Make a category classifier

### ISSUE THREAD

**Issue by strum3nt** [openlawnz/openlawnz-ml#2]
**Problem** 
We need to extract (with reasonable accuracy) the category or categories that apply to a case by analysing the case text.

**Desired Outcome**
A machine learning algorithm that guess whether or not a case belongs to category [x]. An example is the existing crime/not-crime classifier which (on an earlier version of our database) was capable of telling whether a case was a criminal case or not.

**Guide and constraints**
A link to the latest full database in postgres format is available to volunteers - get in touch with @strum3nt or @williamparry if you require a copy.

You may research and select an appropriate ML methodology - we are more concerned about accuracy than the particular method of machine learning being used.

A classifier should relate to a single category. Initial categories are as follows:

1. Crime
2. Family 
3. Resource management
4. Bankruptcy
5. Company liquidation
6. Property (real estate)
7. Commercial

The classifier **must** output a numerical confidence for each case.

**Training data** 
You may need training data (likely, positive and negative sets). There are two ways you may like to obtain training data.

One way is to enlist some of our lawyer and law student volunteers. Talk to the #lawtalk team on Slack. 

The other way is to create a training set using humans by building a Amazon Mechanical Turk or similar system. Funding may be available for this - talk to @strum3nt or @williamparry.
